Description: Improve gui failure description
 The start_gui function only fails when no supported sensors are found;
 update the error description to reflect that.
Author: Nanley Chery <nanleychery@gmail.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.c
+++ b/src/main.c
@@ -123,7 +123,7 @@
 
     /* This will start the GUI. */
     if ( start_gui( argc, argv ) != SUCCESS ) {
-        fprintf( stderr, "GUI failed!\n" );
+        fprintf( stderr, "No supported sensors found!\n" );
     }
 
     /* Clean up the sensors library. */
